pages/recreation.html

```diff
@@ -0,0 +1,50 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        <link rel="stylesheet" href="../stylesheets/style.css">
+        <link rel="stylesheet" href="../stylesheets/recreation.css">
+        <title>Blue Ridge Mountains</title>
+    </head>
+    <body>
+        <main class="container">
+            <header class="header">
+                <h1>Blue Ridge Mountains</h1>
+            </header>
+            <nav class="nav">
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a href="facts.html">Cold Hard Facts</a></li>
```

Comment:
Some indentation is 2-spaces and some is 4-spaces, to make things easy to read and reduce possible confusion, we should pick one and be consistent across the project.